### PR TITLE
Added greet method

### PR DESCRIPTION
**Pull request by MilanPospisil**
This PR will create workflow test if galaxykit app is runnable.

It also adds Greet method that only prints hello. Its difference from help is that it enters client.py, but does not connect to backend.

### CODE REVIEW

File: galaxykit/command.py [hunk 1]
```diff
@@ -452,7 +462,10 @@ def main():
             "username": args.auth_username,
             "password": args.auth_password,
         }
-    client = GalaxyClient(args.server, creds, https_verify=https_verify)
+
+    # greet is here only for test if application is running and without the server
+    if args.kind != "greet":
+        client = GalaxyClient(args.server, creds, https_verify=https_verify)
```
Comment (himdel) [inline galaxykit/command.py:468]: this is the part we need to work .. so we can't skip it here if galaxykit greet is supposed to be used for tests

File: .github/workflows/test_run.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          python-version: "3.8"
+
+      - name: Checking dir
+        run: echo $(ls)
+
+      - name: Update apt
+        run: sudo apt update
+
+      - name: Install pip
+        run: sudo apt install pip
```
Comment (himdel) [inline .github/workflows/test_run.yml:22]: setup-python already handles installing pip

File: .github/workflows/test_run.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        
+      - name: install dependencies
+        run: sudo pip install requests simplejson orionutils pyyaml
+
+      - name: install dependencies_olny_simplejson
+        run: sudo pip install simplejson
```
Comment (himdel) [inline .github/workflows/test_run.yml:28]: we should not be installing anything other than galaxykit

if galaxykit dependencies are declared correctly, pip will install them with galaxykit

File: .github/workflows/test_run.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  
+      - name: Python is running
+        run: python -c 'print("Hi")'
+
+      - name: Galaxykit help is running
+        run: python -c 'from galaxykit.command import main;main();' -u admin -p admin --help
```
Comment (himdel) [inline .github/workflows/test_run.yml:34]: `run: galaxykit -u admin -p admin --help` 

invoking main via import won't necessarily do the same thing

**Comment by himdel** [comment]
Looks good, seems the CI works and is passing now :+1:

But the last commit was attempting to add a missing newline and failed .. looks like you added a newline and a tab .. so it's just a new empty line that's not ending in a newline :) There should be an editor setting to toggle, most linux based editors should do this correctly.

And black complains about the parentheses in command.py ;)

**Comment by MilanPospisil** [comment]
> Looks good, seems the CI works and is passing now 👍
> 
> But the last commit was attempting to add a missing newline and failed .. looks like you added a newline and a tab .. so it's just a new empty line that's not ending in a newline :) There should be an editor setting to toggle, most linux based editors should do this correctly.
> 
> And black complains about the parentheses in command.py ;)

I hope that is all :-)

**Review by himdel — APPROVED** [review]
LGTM :+1:

**Comment by himdel** [comment]
It is now :)

**Pull request merged**